GliaWidgets/Sources/Upload/FileUploader.Environment.Interface.swift

```diff
@@ -7,8 +7,7 @@ extension FileUploader {
         var data: FoundationBased.Data
         var date: () -> Date
         var gcd: GCD
-        var localFileThumbnailQueue: FoundationBased.OperationQueue
-        var uiImage: UIKitBased.UIImage
+        var createThumbnailGenerator: () -> QuickLookBased.ThumbnailGenerator
```

Comment:
I wonder if we are to benefit from multiple instances of `ThumbnailGenerator`. Maybe one will be enough to be shared between cells? Or it does impact performance and causes issue when shared?

Reply:
I've tested using one `QLThumbnailGenerator.shared` instance and it worked significantly worse rather then separate ones